recipes/ad3/recipe/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake  -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+make install
```

Comment:
Is there some sort of `make check` that we can run (since it appears we are building libraries)?